symphony-bdk-bom/build.gradle

```diff
@@ -32,55 +32,55 @@ dependencies {
         api "org.finos.symphony.bdk:symphony-bdk-template-handlebars:$project.version"
 
         // External dependencies
-        api 'org.projectlombok:lombok:1.18.16'
+        api 'org.projectlombok:lombok:1.18.20'
 
         api 'org.apiguardian:apiguardian-api:1.1.0'
 
-        api 'org.slf4j:slf4j-api:1.7.30'
-        api 'org.slf4j:slf4j-log4j12:1.7.30'
+        api 'org.slf4j:slf4j-api:1.7.32'
+        api 'org.slf4j:slf4j-log4j12:1.7.32'
 
-        api 'commons-io:commons-io:2.8.0'
+        api 'commons-io:commons-io:2.11.0'
         api 'commons-codec:commons-codec:1.15'
         api 'commons-beanutils:commons-beanutils:1.9.4'
-        api 'org.apache.commons:commons-lang3:3.11'
+        api 'org.apache.commons:commons-lang3:3.12.0'
         api 'org.apache.commons:commons-text:1.9'
         api 'commons-logging:commons-logging:1.2'
         api 'com.brsanthu:migbase64:2.2'
         api 'io.jsonwebtoken:jjwt:0.9.1'
-        api 'org.bouncycastle:bcpkix-jdk15on:1.67'
+        api 'org.bouncycastle:bcpkix-jdk15on:1.69'
         api 'com.google.code.findbugs:jsr305:3.0.2'
 
-        api 'io.github.resilience4j:resilience4j-retry:1.6.1'
+        api 'io.github.resilience4j:resilience4j-retry:1.7.1'
 
-        api 'com.fasterxml.jackson.core:jackson-databind:2.11.4'
```

Comment:
this is a BOM for Jackson that we could use maybe

Reply:
Yep, done